site2/docs/cookbooks-message-throttling.md

```diff
@@ -0,0 +1,86 @@
+---
+id: message-throttling
+title: Message throttling
+sidebar_label: "Message throttling"
+---
+
```

Comment:
It would be great if we could structure this doc (like a table view ).


```
feature-name
description 
example-case
how-to-config 
throttling-condition
throttling-condition-priority


* throttling-condition-priority:  OR or AND relation with other conditions. If there is a priority order, what's the order?
```

Reply:
Good suggestion, I will try to structure this doc later

Reply:
I added a new table to clarify the features